idaes/core/util/misc.py

```diff
@@ -213,18 +214,48 @@ def svg_tag(
 
 
 # Author: John Eslick
-def copy_port_values(destination, source):
+def copy_port_values(destination=None, source=None, arc=None):
```

Comment:
Very minor comment. Can we switch source and destination arguments to be consistent with arc definition? I am fine with this if you prefer this.

Reply:
Yeah, I want to, but this method has been around for a long time, and I afraid to break peoples stuff.  I guess it's best to just use named args to avoid confusion.

Reply:
Okay, fine with me.